Microsoft.Toolkit.Uwp/Microsoft.Toolkit.Uwp.csproj

```diff
@@ -14,7 +14,7 @@
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
     
     <!-- TODO: Update to later version -->
-    <PackageReference Include="Microsoft.Identity.Client" Version="1.1.2-preview0008" />
```

Comment:
We can remove this reference, I don't believe it's used

Reply:
@rschuerer can you try removing it and seeing what happens?

Reply:
I removed it as it's unnecessary.